R/ff_analyze.R

```diff
@@ -25,72 +25,106 @@
 #'
 #' @export
 
-ff_analyze <- function(predictions,groundtruth,forestmask=NULL, csvfile = NULL, country = NULL,
-                       append = TRUE, analysis_polygons = NULL, return_polygons=TRUE, remove_empty=TRUE,
-                       date=NULL, tile=NULL, method=NA, addWKT = FALSE, verbose=FALSE){
-  if (!(class(predictions) %in% c("character","SpatRaster"))) {stop("predictions is not a raster or path to a raster")}
-  if (!(class(groundtruth) %in% c("character","SpatRaster"))) {stop("predictions is not a raster or path to a raster")}
-  if (!is.null(csvfile)) {if (append == TRUE & !file.exists(csvfile)) {
-    append <- FALSE
-    ff_cat("CSV file did not yet exist, creating empty one", verbose = verbose)
-    }}
+ff_analyze <- function(predictions, groundtruth, forestmask = NULL, csvfile = NULL, country = NULL,
+                       append = TRUE, analysis_polygons = NULL, return_polygons = TRUE, remove_empty = TRUE,
+                       date = NULL, tile = NULL, method = NA, addWKT = FALSE, verbose = FALSE) {
```

Comment:
what is addWKT?

Reply:
addWKT is a method to add the coordinates and class of a spatial dataset as an attribute. it's for vector data (polygons, lines and points). it is used in powerbi